spec/Candid.md

```diff
@@ -864,142 +864,213 @@ service { <methtype>;* } <: service { <methtype'>;* }
 service { <name> : <functype>; <methtype>;* } <: service { <name> : <functype'>; <methtype'>;* }
 ```
 
-### Elaboration
+### Decoding
 
-To define the actual coercion function used during deserialisation of a value, we extend the subtyping relation to a ternary *elaboration* relation `T <: T' ~> f`, where `f` is a suitable coercion function of type `T -> T'`.
+This subtyping is implemented when decoding a Candid value. We provide a ternary relation `V :? T ~> V'` to describe when a value `V` can be decoded at type `T` to a value `V'` of type `T`.
```

Comment:
Nit: The `:?` reads as if this was partial somehow. Why not plain `:`? You could even write it as `v ~> v' : T`, then the colon is on the right side.

Reply:
Oh, `v ~> v' : T` is nice! Will do.